profiles/Base.java

```diff
@@ -25,6 +25,10 @@ public void processFeature(SourceFeature source, FeatureCollector features) {
             }
             feature.setMinZoom(minzoom);
             OvertureProfile.addFullTags(source, feature, MAXZOOM);
+        }  else if (layer.equals("bathymetry")) {
+            feature.setMinPixelSize(0);
```

Comment:
This can result in some pretty big tile sizes because very small polygons aren't generalized away